source/_lovelace/button.markdown

```diff
@@ -7,10 +7,39 @@ description: "The Button card allows you to add buttons to perform tasks."
 The Button card allows you to add buttons to perform tasks.
 
 <p class='img'>
-<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button card'>
-Screenshot of the Button card.
+<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of three Button Cards'>
+Screenshot of three Button Cards.
 </p>
 
+### Card Settings
+
+{% configuration_basic %}
+Entity:
+  description: The entity ID the card interacts with, for example `light.living_room`.
+Name:
+  description: The button name that is displayed on the card. If this field is left blank and the card interacts with an entity, the button name defaults to the entity name. Otherwise, no name is displayed.
+Icon:
+  description: The icon that is displayed on the card. If this field is left blank and the card interacts with an entity, the icon defaults to the entity domain icon. Otherwise, no icon is displayed.
+Show Name:
+  description: A toggle to show or hide the button name.
+Show Icon:
+  description: A toggle to show or hide the icon.
+Icon Height:
+  description: The height of the icon, in pixels.
+Theme:
+  description: The card theme, which may be set to any installed theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
+Tap Action:
+  description: The action taken on card tap. For more information, see the [action documentation](/lovelace/actions/#tap-action).
+Hold Action:
+  description: The action taken on card tap and hold. For more information, see the [action documentation](/lovelace/actions/#hold-action).
+{% endconfiguration_basic %}
+
+## Code Editor or YAML mode
+
+Use these configuration variables if you prefer to use the Code Editor to configure your cards, or if you use YAML mode.
```

Comment:
```suggestion
Use these configuration variables if you prefer to use the Code Editor to configure your cards or use YAML mode.
```

Reply:
Meh... not sure either... if you.. if you.. felt a bit weird...

Reply:
What about, “Use these YAML configuration variables if you are setting up your cards in the Code Editor or in YAML mode.”